pandas/core/frame.py

```diff
@@ -1253,11 +1257,17 @@ def to_excel(self, excel_writer, sheet_name='Sheet1', na_rep='',
         """
         from pandas.io.excel import ExcelWriter
         if self.columns.nlevels > 1:
-            raise NotImplementedError("Writing as Excel with a MultiIndex is "
-                                      "not yet implemented.")
+            if not index:
+                raise NotImplementedError("Writing to Excel with MultiIndex"
+                                          " columns and no index is not yet"
```

Comment:
can you change 'and no index' to 'and with 'index=False' ' (otherwise you could understand that the dataframe has no index)